test-scripts/testemail.ts

```diff
@@ -0,0 +1,26 @@
+import { OpportunityEmail } from '@/components/resend-emails/recruiter-dev-enquiry';
+import { env } from '@/env';
+import { Resend } from 'resend';
+
+const resend = new Resend(env.RESEND_API_KEY);
+const emailSendResponse = await resend.emails.send({
+  from: 'hello@overpoweredjobs.com',
+  to: 'ajinkyabodke@gmail.com',
+  subject: 'hello world',
+  react: OpportunityEmail({
+    devName: 'AjinkyaB',
+    message: 'Hi,testing 123',
+    recruiterName: 'Sid 100x',
+    quotePriceInRupees: 10000,
+    typeOfWork: 'freelance',
+    recruiterEmail: 'siddhant@100xengineers.com',
```

Comment:
Why are we hardcoding these values?

Reply:
Well,these scripts were just added to test resend,would remove them